File: app/src/androidTest/java/com/anthonysottile/kenken/test/SignTests.java
package com.anthonysottile.kenken.test;

import com.anthonysottile.kenken.Sign;

import junit.framework.TestCase;

public class SignTests extends TestCase {

    public void testGetIntValue() {
        assertEquals(Sign.Add.getIntValue(), 0);
        assertEquals(Sign.Subtract.getIntValue(), 1);
        assertEquals(Sign.Multiply.getIntValue(), 2);
        assertEquals(Sign.Divide.getIntValue(), 3);
        assertEquals(Sign.None.getIntValue(), 4);
    }

    public void testToString() {
        assertEquals(Sign.Add.toString(), "+");
        assertEquals(Sign.Subtract.toString(), "-");
        assertEquals(Sign.Multiply.toString(), "*");
        assertEquals(Sign.Divide.toString(), "/");
        assertEquals(Sign.None.toString(), "");
    }
}

File: app/src/androidTest/java/com/anthonysottile/kenken/test/SignTests.kt
package com.anthonysottile.kenken.test

import com.anthonysottile.kenken.Sign

import junit.framework.TestCase

class SignTests : TestCase() {

    fun testGetIntValue() {
        assertEquals(Sign.Add.intValue, 0)
        assertEquals(Sign.Subtract.intValue, 1)
        assertEquals(Sign.Multiply.intValue, 2)
        assertEquals(Sign.Divide.intValue, 3)
        assertEquals(Sign.None.intValue, 4)
    }

    fun testToString() {
        assertEquals(Sign.Add.toString(), "+")
        assertEquals(Sign.Subtract.toString(), "-")
        assertEquals(Sign.Multiply.toString(), "×")
        assertEquals(Sign.Divide.toString(), "÷")
        assertEquals(Sign.None.toString(), "")
    }
}
